Tests/AK/TestBitmap.cpp

```diff
@@ -286,6 +331,25 @@ TEST_CASE(find_one_anywhere_edge_case)
     }
 }
 
+RANDOMIZED_TEST_CASE(find_one_anywhere)
+{
+    GEN(init, Gen::boolean());
+    GEN(size, Gen::unsigned_int(1, 64));
+    GEN(hint, Gen::unsigned_int(size - 1));
+
+    GEN(new_value, Gen::boolean());
+    GEN(i, Gen::unsigned_int(size - 1));
+
+    auto bitmap = MUST(Bitmap::create(size, init));
+    bitmap.set(i, new_value);
+
+    Optional<size_t> result = new_value
+        ? bitmap.find_one_anywhere_set(hint)
+        : bitmap.find_one_anywhere_unset(hint);
+
+    EXPECT(result.has_value());
```

Comment:
We have, roughly, three choices:

1. keep all the parameters completely random and don't check the specific index that's found and returned (since if you're setting `true` in an already fully `true` bitmap, the index that will be returned is 0, not the `i` at which you have set)
2. hardcode some of the parameters (ie. if new_value = !init) and be able to check the index since it will always be the generated `i`
3. like 1, randomize everything, but be a bit smart about what index you're expecting: if `new_value == init` then expected index == 0, else expected index == `i`.

Do you want (3)?

Reply:
(3) would be great, yes.